src/network/networkd-queue.h

```diff
@@ -68,9 +72,15 @@ typedef struct Request {
                 void *object;
         };
         void *userdata;
-        unsigned *message_counter;
-        link_netlink_message_handler_t netlink_handler;
-} Request;
+
+        /* incremented when requested, decremented when request is completed or failed. */
+        unsigned *counter;
+        /* called in netlink handler, the 'counter' is decremented before this is called.
```

Comment:
It would be good to have this explanation in the commit message - it's a large change, and the functional part is only explained here, halfway through the commit